lib/controller.dart

```diff
@@ -349,14 +362,23 @@ void command(UIAction action, Data data) {
     case UIAction.changeAnalysisTab:
       var d = data as AnalysisTabChangeData;
       _selectedAnalysisTabIndex = d.tabIndex;
+      _activeFilters = {};
+      _filterValues = {};
+      _comparisonFilterValues = {};
+      view.removeFiltersWrapper();
+      view.removeAllChartWrappers();
+      _computeFilterDropdownsAndRender();
+      _computeChartBucketsAndRender();
```

Comment:
what's the performance of clearing up the UI and reloading everything? is there any flickering? If yes, we might need to look at incrementally updating the UI...

Reply:
At the moment I don't see flickers as such. 

But if I have to build this in..
1. The charts need to be taken out as they are highly likely to be different values for each of the tabs. Also the chartjs doesn't refresh the "on-air" image symbols without throwing away the canvas and re-rendering, say when you want to toggle on / off comparison.
2. For the filter dropdowns, I was not able to maintain the order of filter keys as I could only append them in the end after comparing the diff of filter keys across both tabs. Just realised as I was writing this I could use hidden parameter against each of the filter rows. - Will update this PR with this functionality soon.